src/Futhark/Analysis/Pretty.hs

```diff
@@ -0,0 +1,73 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Futhark.Analysis.Pretty
+  ( prettyTuple,
+    prettyTupleLines,
+    prettyString,
+  )
+where
+
+import Data.IntMap.Strict qualified as S
+import Data.Map.Strict qualified as M
+import Futhark.Analysis.AccessPattern
+import Futhark.Util.Pretty
+
+instance Pretty (IndexTable rep) where
```

Comment:
Put these instances in the AccessPattern module itself.

Reply:
Fixed in [99f26a27](https://github.com/diku-dk/futhark/pull/2027/commits/99f26a270718fd9e8bb3804a136054101a741116)